cli/tests/nonce.rs

```diff
@@ -66,6 +66,7 @@ fn test_nonce() {
     config_payer.command = CliCommand::CreateNonceAccount {
         nonce_account: read_keypair_file(&keypair_file).unwrap().into(),
         lamports: 1000,
+        authorized_noncer: None,
```

Comment:
How about a test to verify that `authorized_noncer: Some(`

Reply:
Ah, yeah!  I guess I just did one for the parser.  Will add